superset/views/database/api.py

```diff
@@ -145,14 +146,14 @@ class DatabaseRestApi(DatabaseMixin, BaseSupersetModelRestApi):
 
     openapi_spec_tag = "Database"
 
-    @expose(
-        "/<int:pk>/table/<string:table_name>/<string:schema_name>/", methods=["GET"]
-    )
+    @expose("/<int:pk>/table/<table_name>/<schema_name>/", methods=["GET"])
```

Comment:
Note per [here](https://flask.palletsprojects.com/en/1.1.x/quickstart/#variable-rules) `string` is the default converter type and thus doesn't need to be explicitly defined. This is consistent with other views.